### PR TITLE
[WIP] V2 plugins

### PR DESCRIPTION
**Pull request by angstwad**

### CODE REVIEW

File: plugins/connection/sshbb.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # (c) 2012, Michael DeHaan <michael.dehaan@gmail.com>
```
Comment (j2sol) [inline plugins/connection/sshbb.py]: Is this just upstream's ssh plugin, with our minor change integrated?

File: requirements.txt [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
-python-cinderclient
-oslo.utils
 tox
 pep8
--e git://github.com/blueboxgroup/ansible.git@1.9.2-bbg#egg=ansible
--e git://github.com/blueboxgroup/ursula-cli.git#egg=ursula-cli
--e git://github.com/openstack-infra/shade.git@ab1c566cb8afbf477492180e2cc8257817a06893#egg=shade
+shade
+oslo.utils
+ansible<=2.0.2.0
```
Comment (j2sol) [inline requirements.txt]: Can we changes his to just be <=3.0  ?

File: v1_plugins/action/sensu_check.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
+from ansible.utils import parse_kv, template
```
Comment (j2sol) [inline v1_plugins/action/sensu_check.py]: Is there a reason to leave the v1_plugins around in the tree?

File: plugins/connection/sshbb.py (hunk 1 above)
Comment (angstwad) [inline plugins/connection/sshbb.py]: Yes.

File: requirements.txt (hunk 2 above)
Comment (angstwad) [inline requirements.txt]: I think actually it needs to be `<=2.0.2.0,>3.0`

2.0.2.0 comes with a big bug fix that I ran into quite a lot: https://github.com/ansible/ansible/issues/13876

File: v1_plugins/action/sensu_check.py (hunk 3 above)
Comment (angstwad) [inline v1_plugins/action/sensu_check.py]: v1 will go away when this is ready for final review

File: requirements.txt (hunk 2 above)
Comment (j2sol) [inline requirements.txt]: That's fair, I just want the upper bound to be 3.0.

**Comment by j2sol** [comment]
retest

**Pull request closed without merging**